oat-sa/extension-tao-itemqti
# [NEX-1456] Add possibility to select custom response processing in test item authoriding

Related to : https://oat-sa.atlassian.net/browse/NEX-1456
 
Add possibility to select custom response processing type in case if [xmlEditRp](https://github.com/oat-sa/extension-tao-xmledit-responseprocessing) installed
  
#### How to test
 
- prepare an instance of TAO with xmlEditRp extension
- create an item
- navigate to item authoring
- try to set up response processing for the interaction
- make sure that it is possible to select custom response processing
- after selected custom response processing make sure that all the interaction switch to this type
- try to add an interaction and make sure that newly added interactions has custom response processing
- check the default value of custom response processing
